designcompose/src/main/java/com/android/designcompose/InteractionState.kt

```diff
@@ -435,7 +437,9 @@ internal fun InteractionState.dispatch(
                                     key,
                                     action.destination_id.get(),
                                     undoInstanceId,
-                                    action.transition.get(),
+                                    SmartAnimateTransition(
```

Comment:
Oh - this is a nice way to convey the kind of transition the animation system should do. Thanks for aligning both paths here!